src/arena.rs

```diff
@@ -214,6 +214,10 @@ impl<T> Arena<T> {
         self.nodes.iter()
     }
 
+    pub fn iter_mut(&mut self) -> impl Iterator<Item = &mut Node<T>> {
```

Comment:
Can you please add a doc comment for this? :)

Reply:
Do you want the doc with example ? :]

Reply:
That would be wonderful!